relay-dynamic-config/src/project.rs

```diff
@@ -123,4 +127,11 @@ pub struct LimitedProjectConfig {
     pub features: BTreeSet<Feature>,
     #[serde(skip_serializing_if = "Vec::is_empty")]
     pub tx_name_rules: Vec<TransactionNameRule>,
+    /// Whether or not a project is ready to mark all URL transactions as "sanitized".
+    #[serde(skip_serializing_if = "is_false")]
```

Comment:
nit: we can avoid creating the `is_false` function with [`Not::not`](https://doc.rust-lang.org/std/ops/trait.Not.html#tymethod.not), but I don't know how easy to understand it'd be.